BEIMA.Backend.FT/BeimaClient.cs

```diff
@@ -194,7 +194,7 @@ public async Task<HttpResponseMessage> SendMultiPartRequest(string route, Object
         /// <returns>The device with the given id.</returns>
         public async Task<Device> GetDevice(string id)
         {
-            var response = await SendRequest($"api/device/{id}", HttpVerb.GET);
+            var response = await SendRequest($"device/{id}", HttpVerb.GET);
             var content = await response.Content.ReadAsStringAsync();
```

Comment:
I'm not sure the api part was removed, but just the beima part was added

Reply:
When I run it on my machine it seems to be replaced? 
![image](https://user-images.githubusercontent.com/43118853/164990900-4da4d968-48ce-41a2-98b1-8e5082f06f41.png)

Reply:
Ah okay, good to know

Reply:
Looks like I just needed to add in an extra slash was the problem